feature_layers/feature-collection-layer-from-portal/README.md

```diff
@@ -0,0 +1,34 @@
+# Feature Collection Layer From Portal
+
+This sample demonstrates opening a feature collection saved as a portal item
+
+![](FeatureCollectionLayerFromPortal.png)
```

Comment:
We've been trying to populate the 'alt' text for the screenshot so it's consistent and accessible:

```suggestion
![Image of create feature collection layer portal item](FeatureCollectionLayerFromPortal.png)
```